backend/src/services/update-user/update-user.service.ts

```
@@ -82,6 +82,26 @@ const deleteUser = async (id: string): Promise<Partial<UpdateUserReq>> => {
   };
 };
 
+const getUser = async (id: string): Promise<Partial<UpdateUserReq> | null> => {
+  const user = await prisma.user.findFirst({
+    where: {
+      id,
+    },
+  });
+
+  return user
+    ? {
+        name: user.name,
+        birthYear: user.birth_year,
+        sex: user.sex,
+        phone: user.phone,
+        email: user.email,
+        location: user.location,
+        photoUrl: user.photo_url,
+      }
+    : null;
```

I would rather throw an exception here instead of return ambiguous `null` value